data/data/vsphere/variables-vsphere.tf

```diff
@@ -76,4 +76,3 @@ variable "vsphere_control_plane_num_cpus" {
 variable "vsphere_control_plane_cores_per_socket" {
   type = number
 }
-
```

Comment:
Superfluous change.

Reply:
Just wanted to check where the code fails. I'll fix all these fmt errors when I'm a little more confident the code would work. Sorry about the mess :-)